.gitignore

```diff
@@ -20,3 +20,4 @@ msbuild.log
 # Local constant file
 tests/constants.local.ps1
 
+functions/Copy-DbaResourceGovernor_TEST.ps1
```

Comment:
@suxstellino you need to remove this update from the PR

Reply:
done, pushing